CMakeLists.txt

```diff
@@ -152,6 +152,21 @@ if(CMAKE_SYSTEM_NAME STREQUAL "AIX")
   list(APPEND uv_sources src/unix/aix.c)
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "OS400")
+  list(APPEND uv_defines
+       _ALL_SOURCE
+       _LINUX_SOURCE_COMPAT
+       _THREAD_SAFE
+       _XOPEN_SOURCE=500)
+  list(APPEND uv_sources
+    src/unix/aix-common.c
+    src/unix/ibmi.c
+    src/unix/posix-poll.c
+    src/unix/no-fsevents.c
+    src/unix/no-proctitle.c
+    )
+endif()
+
```

Comment:
I'm guessing this should go down after OpenBSD to be in sorted order.

Reply:
Thanks for noticing the alphabetization! Slid it down.